docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: "3.8"
+
+services:
+  localstack:
+    container_name: localstack
+    image: localstack/localstack:1.3.0
+    ports:
+      - "127.0.0.1:5001:5001"
+    volumes:
+      - "./docker/uid2-core/src/init-aws.sh:/etc/localstack/init/ready.d/init-aws-core.sh"
+      - "./docker/uid2-core/src/s3/core:/s3/core"
+      - "./docker/uid2-core/src/kms/seed.yaml:/init/seed.yaml"
+      - "./docker/uid2-optout/src/init-aws.sh:/etc/localstack/init/ready.d/init-aws-optout.sh"
+      - "./docker/uid2-optout/src/s3/optout:/s3/optout"
+    environment:
+      - EDGE_PORT=5001
+      - KMS_PROVIDER=local-kms
+    healthcheck:
+      test: awslocal s3api wait bucket-exists --bucket test-core-bucket
+        && awslocal s3api wait bucket-exists --bucket test-optout-bucket
+      interval: 5s
+      timeout: 5s
+      retries: 3
+
+  core:
+    container_name: core
+    image: ghcr.io/iabtechlab/uid2-core:<CORE_VERSION>
+    ports:
+      - "127.0.0.1:8088:8088"
+      - "127.0.0.1:9088:9088"
+      - "127.0.0.1:5088:5005"
+    volumes:
+      - ./docker/uid2-core/conf/default-config.json:/app/conf/default-config.json
+      - ./docker/uid2-core/conf/local-e2e-docker-config.json:/app/conf/local-config.json
+    depends_on:
+      localstack:
+        condition: service_healthy
+    healthcheck:
+      test: wget --tries=1 --spider http://localhost:8088/ops/healthcheck || exit 1
+      interval: 10s
+
+  optout:
+    container_name: optout
+    image: ghcr.io/iabtechlab/uid2-optout:<OPTOUT_VERSION>
+    ports:
+      - "127.0.0.1:8090:8081"
+      - "127.0.0.1:5090:5005"
+    volumes:
+      - ./docker/uid2-optout/conf/run_tool_local_e2e.sh:/app/run_tool.sh
+      - ./docker/uid2-optout/conf/default-config.json:/app/conf/default-config.json
+      - ./docker/uid2-optout/conf/local-e2e-docker-config.json:/app/conf/local-config.json
+      - ./docker/uid2-optout/mount/:/opt/uid2/optout/
```

Comment:
I don't think we actually need to mount this volume, might be okay to remove this line

Reply:
The `./docker/uid2-optout/mount/:/opt/uid2/optout/` is actually needed, but we can remove `./docker/uid2-optout/conf/run_tool_local_e2e.sh:/app/run_tool.sh`. Removed and tested: https://github.com/IABTechLab/uid2-operator/actions/runs/7202448532/job/19620584175